Tools/visualize_log.py

```
@@ -1,221 +1,212 @@
 
-    # Print total number of events.
-    total_events = len(event_df)
-    print(f"Total number of events: {total_events}")
+    telemetry_file_path = FLAGS.telemetry_file
+    event_file_path = FLAGS.event_file
+    if not telemetry_file_path or not event_file_path:
+        telemetry_file_path = _find_latest_telemetry_file()
+        event_file_path = _find_latest_event_log()
+    if not telemetry_file_path or not event_file_path:
+        raise ValueError(
+            "Both a telemetry file and an event log must be provided.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)